### PR TITLE
Julia implementation

### PR DESCRIPTION
**Pull request by reagent**
Less code than my C version ... go figure.

### CODE REVIEW

File: languages/Julia/encrypt.jl [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+if (!is_hex_string(ARGS[1]))
+  println("Key must be a hexadecimal string")
+  exit(2)
+end
+
+key, key_index      = ARGS[1], 1
+message, ciphertext = read_from(STDIN), ""
+
+for message_index = 1:endof(message)
```
Comment (nhunzaker) [inline languages/Julia/encrypt.jl]: I love this

Comment (reagent) [inline languages/Julia/encrypt.jl]: Yea, `endof` makes iteration over strings pretty easy.

**Comment by dce** [comment]
:+1: looks really nice.

**Comment by reagent** [comment]
thanks bors

**Pull request merged**